listenbrainz/webserver/static/js/src/stats/UserArtistMap.tsx

```diff
@@ -110,8 +110,10 @@ export default class UserArtistMap extends React.Component<
   getData = async (): Promise<UserArtistMapResponse> => {
     const { range, user } = this.props;
     try {
-      const data = await this.APIService.getUserArtistMap(user.name, range);
-      return data;
+      return await this.APIService.getUserArtistMap(
+        user ? user.name : undefined,
```

Comment:
I'm a fan of this shorter version which safely accesses a nested property, and returns undefined otherwise
This applies to other instances too:
```suggestion
        user?.name,
```

Reply:
Makes sense. fixed.